### ISSUE TITLE
Steam version not working properly.

### ISSUE THREAD

**Issue by coppolaemilio** [godotengine/godot#16515]
Hello there, I have updated the Steam version of Godot this morning (it should be the release 3.0) but I'm getting an error on launch:

![steamerror](https://user-images.githubusercontent.com/2206700/36032384-1bb36530-0dae-11e8-986c-7e84c01868e4.PNG)

**OS/device including version:**
Windows 10

**Comment by AurumStein** [comment]
Yep the 64 bit executable seems missing, I downloaded the one of the site and placed it in the folder and renamed it as a temporary fix.

**Comment by akien-mga** [comment]
Steam seems buggy, it doesn't properly deploy the 64-bit binary for some reason..
